modules/ducktests/tests/ignitetest/services/utils/templates/cache_macro.j2

```diff
@@ -22,10 +22,20 @@
                 {% for cache in caches %}
                 <bean class="org.apache.ignite.configuration.CacheConfiguration">
                     <property name="name" value="{{ cache.name }}"/>
+                    <property name="cacheMode" value="{{ cache.cache_mode or 'PARTITIONED' }}"/>
                     {% if cache.cache_mode == 'PARTITIONED' %}
-                        <property name="backups" value="{{ cache.backups or 0 }}"/>
+                    <property name="backups" value="{{ cache.backups or 0 }}"/>
```

Comment:
ASAIK you may set backups at each mode, but replicated will ignore of fail.
please simplify this.

Reply:
fixed

Reply:
fixed